### DISCUSSION TITLE
Problem with flashing x299

### DISCUSSION THREAD

**Discussion by timido22** [xCuri0/ReBarUEFI#181] [Q&A]
Hi, I may have a lot of stupid questions today, but I can't seem to understand. :( 
I can't patch 
![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/62a55d23-3d39-4e3e-8615-8b66069a7b42)

When I try flashing the BIOS through ASRock Instant Flash, I get an error "Secure Flash check fail."

What I should do?

p.s. Asrock x299 Steel Legend

**Comment by Garithos** [comment]
Use UBU to remove the secure flash check
https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627

**Reply by timido22** [reply]
Hi, thx. I will try. Can you look on this https://github.com/xCuri0/ReBarUEFI/discussions/182 discussion. I found interesting things. If I try to patch bios with letter 'L'. I get a error. 

Bios with "L" - with enabled C.A.M.(Clever Access Memory)
